CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing to the Pet Store Demo
+
+## Prerequisites
+
+Before getting started, you should install some developer tools.
+
+1. [git][get-git] to manage source code
+1. [Docker][get-docker] to manage dependencies and runtime environments
+
+[get-docker]: https://docs.docker.com/engine/installation
+[get-git]: https://git-scm.com/downloads
+
+## Building
+Run `./bin/build`. Requires Docker.
+
+## Testing
+To test against a specific database type, run `./test/test {db type}`, where
+`db type` is `mysql`, `postgres`, or `mssql`.
```

Comment:
this might be a useful change to make make **db type** stand out from its list of possible values
```suggestion
**db type** is `mysql`, `postgres`, or `mssql`.
```